.github/workflows/graalvm-base.yml

```diff
@@ -57,13 +60,16 @@ jobs:
       - name: "🛠 Build with Gradle"
         id: gradle
         run: |
-          ./gradlew check --no-daemon --continue
+          if ./gradlew tasks --no-daemon --all | grep -w "nativeTest"
+          then
+            ./gradlew nativeTest --no-daemon --continue
+          fi
```

Comment:
I think @melix once said there was a better way to do this

Reply:
We can basically remove this once every project has moved out of the legacy `nativeTest` task, which should be the case by now?

Reply:
We can't remove that since not all modules in a project have `nativeTest` task. So far we had dependency between the `check` and `nativeTest` gradle tasks and we when we executed the `check` task we didn't have to worry whether module has 'nativeTest` task or not, but now we want to remove that dependency and to execute only `nativeTest` tasks in this workflow.